### ISSUE TITLE
Refresh Tokens

### ISSUE THREAD

**Issue by ErnieDaTryHard** [SteamRE/DepotDownloader#500]
### What problem is this feature trying to solve?

The Steam API has support for JWT refresh tokens so we don't have to put in a Steam Guard code for each subsequent login. This is useful programmatically if you don't want to prompt the user every time for a Steam Guard code, just only once.

### How would you like it to be solved?

This could be solved by adding it as an argument to the CLI: -refresh_token [token]

### Have you considered any alternative solutions

Making DepotDownloader myself with support for refresh tokens seems like a major waste of time, and is more likely to be prone to bugs than this project.

### Additional Information

I appreciate this is pretty niche, if it could be implemented that would be amazing.

**Comment by yaakov-h** [comment]
How is this different to the CLI argument `-remember-password`?

**Comment by ErnieDaTryHard** [comment]
> How is this different to the CLI argument `-remember-password`?

It would be a "nice to have" if we can access the refresh token directly in the CLI if we have already gotten it through other means, rather than an account.config file that I can't really use.
